.github/workflows/consumer-acceptance.yml

```
@@ -0,0 +1,171 @@
+jobs:
+  accept:
+    runs-on: ubuntu-24.04
+    timeout-minutes: 30
+    steps:
+      - name: Validate authority-owned orchestration and candidate identity
+        shell: bash
+        env:
+          CALLER_REPOSITORY: ${{ github.repository }}
+          CALLER_SHA: ${{ github.sha }}
+          CALLER_REF_PROTECTED: ${{ github.ref_protected }}
+          AUTHORITY_REPOSITORY: ${{ job.workflow_repository }}
```

actionlint (1.7.12)

[error] property "workflow_repository" is not defined in object type {check_run_id: number; container: {id: string; network: string}; services: {string => {id: string; network: string; ports: {string => string}}}; status: string}

(expression)